gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Gherkin/IgnoreException.cs

```diff
@@ -40,9 +40,6 @@ private static string GetMessage(IgnoreReason reason)
             string reasonSuffix = null;
             switch (reason)
             {
-                case IgnoreReason.LambdaNotSupported:
```

Comment:
Can we replace it with a comment, like `// Include detailed reason for each enum here`?